lib/src/main/kotlin/net/dean/jraw/references/FlairReferences.kt

```diff
@@ -52,7 +51,16 @@ sealed class FlairReference(
      * @see SubredditReference.userFlairOptions
      */
     @EndpointImplementation(Endpoint.POST_SELECTFLAIR)
-    abstract fun updateTo(templateId: String, text: String?)
+    abstract fun updateToTemplate(templateId: String, text: String?)
+
+    /**
+     * Similar to [updateToTemplate] but sets the CSS class directly instead of using a template.
+     * The authenticated user must be a moderator of the [subreddit].
+     */
+    @EndpointImplementation(Endpoint.POST_FLAIR)
+    abstract fun updateToCssClass(cssClass: String, text: String?)
+
+    fun updateToCssClass(cssClass: String) = updateToCssClass(cssClass, null)
```

Comment:
Went ahead and added implementation tests. Only requires to create a subreddit and set it in the test file. In future, we could automate it by having a single modded subreddit as a SharedObject constant value, instead of having a subreddit string everywhere.

Tests are in https://github.com/mattbdean/JRAW/pull/207/commits/83569dc340b4a257015f594b1179f3378e3bc5f0